index.js

```diff
@@ -332,3 +332,30 @@ export const breadcrumbs = {
+  indeterminate: `before:i-text-$color-text-inverted before:text-center before:font-bold before:content-['–'] peer-indeterminate:before:i-border-$color-checkbox-border-active peer-indeterminate:before:i-bg-$color-checkbox-background-active peer-indeterminate:hover:before:i-border-$color-checkbox-border-hover peer-indeterminate:hover:before:i-bg-$color-checkbox-background-active-hover`,
+  labelDisabled: 'pointer-events-none',
+  checkbox: 'before:rounded-2 hover:before:i-border-$color-checkbox-border-hover hover:before:i-bg-$color-checkbox-background-hover',
+  checkboxChecked: 'peer-checked:before:i-border-$color-checkbox-border-active peer-checked:before:i-bg-$color-checkbox-background-active peer-checked:peer-hover:before:i-border-$color-checkbox-border-active-hover peer-checked:peer-hover:before:i-bg-$color-checkbox-background-active-hover',
+  checkboxInvalid: 'before:i-bg-$color-checkbox-negative-background hover:before:i-bg-$color-checkbox-negative-background-hover before:i-border-$color-checkbox-negative-border hover:before:i-border-$color-checkbox-negative-border-hover peer-checked:before:i-bg-$color-checkbox-negative-background-active peer-checked:peer-hover:before:i-bg-$color-checkbox-negative-background-active-hover peer-checked:peer-hover:before:i-border-$color-checkbox-negative-border-active-hover',
+  labelCheckboxColors: 'i-bg-$color-checkbox-background i-border-$color-checkbox-border',
+  radio: 'before:rounded-full peer-checked:before:i-border-$color-radio-border-active peer-checked:before:border-[6] peer-checked:peer-hover:before:i-border-$color-radio-border-active-hover peer-hover:before:i-border-$color-radio-border-hover peer-hover:before:i-bg-$color-radio-background-hover',
+  radioInvalid: 'before:i-bg-$color-radio-negative-background peer-hover:before:i-bg-$color-radio-negative-background-hover before:i-border-$color-radio-negative-border peer-hover:before:i-border-$color-radio-negative-border-hover peer-checked:before:i-border-$color-radio-negative-border-active peer-checked:peer-hover:before:i-border-$color-radio-negative-border-active-hover ',
+  labelRadioColors: 'i-bg-$color-radio-background i-border-$color-radio-border',
+  segmentControl: 'inline-flex relative font-bold rounded-8',
+  scLabel: 'peer-hover:i-bg-$color-buttongroup-background-hover peer-checked:i-text-$color-buttongroup-text-active peer-checked:i-bg-$color-buttongroup-background-active peer-checked:peer-hover:i-bg-$color-buttongroup-background-active-hover peer-checked:i-border-$color-buttongroup-border-active block relative text-14 font-bold pb-4 cursor-pointer i-text-$color-buttongroup-text text-center i-bg-$color-background border-2 i-border-$color-buttongroup-border py-10 px-14 group-first-of-type:rounded-tl-8 group-first-of-type:rounded-bl-8 group-last-of-type:rounded-tr-8 group-last-of-type:rounded-br-8 group-not-last-of-type:border-r-0 ',
+  scJustified: 'flex',
+  scLabelJustified: 'grow-0 shrink-0 basis-auto',
```

Comment:
I think this does the same job as`inline-flex` does on the parent. If this was meant to make all items take equal width, I think we might need to set this to `flex-1` or at least `grow-1`
When the other day I was playing around with the equalWidth variant of the Toggle radio group to make all buttons take equal space, I also noticed that setting `flex` on the parent didn't override `inline-flex`, so those two maybe need to be set conditionally, too. Let me know if you want to look at it together 😊